client/scss/utilities/_root-scope-classes.scss

```diff
@@ -18,6 +30,19 @@
   margin-bottom: $spacing-unit * 2;
 }
 
+.l-margin-s {
+  margin-left: $spacing-unit * 2;
+}
+
+.r-margin-s {
+  margin-right: $spacing-unit * 2;
+}
+
+.h-center {
```

Comment:
I use these all the time - it feels like having them abstracted and then able to compose my components is an easier way to do this.

Component styles should be styles that are specialist for that component only.